ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_bgp.py

```diff
@@ -246,7 +246,14 @@ def _router_bgp_vrfs(self) -> list | None:
                 for bgp_peer in vrf["bgp_peers"]:
                     peer_ip = bgp_peer.pop("ip_address")
                     address_family = f"address_family_ipv{ipaddress.ip_address(peer_ip).version}"
-                    neighbor = {"ip_address": peer_ip, "activate": True}
+                    neighbor = strip_empties_from_dict(
```

Comment:
probably worth adding a comment here to explain why we pop the prefix lists out for posterity

Reply:
added it at the top since we pop some other things in here.